.github/workflows/release.yaml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        if: ${{ !startsWith(github.ref, 'refs/tags/v') }}
+        run: echo "flags=--snapshot" >> $GITHUB_ENV
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20.12'
```

Comment:
I'd rather use `go-version-file: 'go.mod'`
You don't need to update workflow in case of go patch version bump.

Reply:
good idea.

Reply:
Updated in [6ab9a0b](https://github.com/databus23/helm-diff/pull/517/commits/6ab9a0b7b277b254a436dc8ffda8dbc5cbc0754a). Thank you very much for the suggestion ☺️